docs/releases/3.1.txt

```diff
@@ -475,12 +475,16 @@ Miscellaneous
   ``django-admin``.
 
 * The ``HttpRequest.is_ajax()`` method is deprecated as it relied on a
-  jQuery-specific way of signifying AJAX calls, while the current usage tends
-  to use the JavaScript `Fetch API
-  <https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API>`_. Depending on
-  your use case, you can either write your own AJAX detection method, or use
-  the new :meth:`.HttpRequest.accepts` method if your code depends on the
-  client ``Accept`` HTTP header.
+  jQuery-specific way of signifying AJAX calls, while current usage tends to
+  use the JavaScript `Fetch API
+  <https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API>`_.
+  Depending on your use case, you can either write your own AJAX detection
+  method, or use the new :meth:`.HttpRequest.accepts` method if your code
+  depends on the client ``Accept`` HTTP header.
```

Comment:
Please revert all unnecessary changes in the existing description.

Reply:
I fixed the phrase "while the current usage tends to use" which doesn't really make sense,  the rest is diff noise from re-wrapping

Reply:
I missed that you removed `the`.